imports/ui/Body.js

```diff
@@ -15,26 +14,31 @@ import './Index.js';
 import './HongKongNewGame.js';
 import './JapaneseNewGame.js';
 
-Template.JapaneseRanking.helpers({
-	japanesePlayers() {
-		return Players.find({}, {sort: { japaneseElo: -1}});
-	},
+var index;
+
+function initPlayer(key) {
```

Comment:
Would it be better to specify what sort of sorting we want if we want to sort by other metrics layer?